docs/lib/storage/fragments/ios/configureaccess/50_customization.md

```diff
@@ -0,0 +1,136 @@
+### Customize Object Key Path 
+
+You can customize your key path by defining a prefix resolver:
+
+```swift
+import Amplify
+import AmplifyPlugins
+
+struct MyDeveloperDefinedPrefixResolver: AWSS3StoragePluginPrefixResolver {
```

Comment:
Can we add some comments on what the code is doing? The documentation starts off with a struct `MyDeveloperDefinedPrefixResolver` without giving much info about why we need it, I think it would be better if we explain what resolver are and how we use them. Then show the code?

Reply:
Will iterate on this, and add some more comments first. I think ihis is actually subsection of a larger doc that has some content about it above it